src/test/skript/tests/regressions/7471-is tagged or list.sk

```diff
@@ -0,0 +1,6 @@
+test "is tagged or list":
+	register a custom item tag named "or_list_tag_1" using oak log
+	register a custom item tag named "or_list_tag_2" using spruce log
+
+	loop 10 times:
+		assert oak log is tagged as custom tag "or_list_tag_1" or custom tag "or_list_tag_2"
```

Comment:
I believe the original issue was relating to `tag A or B`?!?!
Maybe add another check:
`assert oak log is tagged as custom tag "or_list_tag_1" or  "or_list_tag_2"`

I could be totally wrong here.

Reply:
Well that code works as intended, they should be using tag a or tag b if there want to compare multiple tags

Reply:
That's fair yeah.